tests/testthat/test-epidist_db.R

```diff
@@ -34,21 +21,66 @@ test_that("epidist_db works as expected with author specified", {
 
   expect_s3_class(edist, class = "epidist")
   expect_length(edist, 9)
-  expect_named(
-    edist,
-    c("disease", "epi_dist", "prob_dist", "uncertainty", "summary_stats",
-      "citation", "metadata", "method_assess", "notes")
+  expect_s3_class(validate_epidist(edist), class = "epidist")
+})
+
+test_that("epidist_db works as expected with subsetting", {
+  # suppress message about citation
+  edist <- suppressMessages(
+    epidist_db(
+      disease = "influenza",
+      epi_dist = "incubation_period",
+      subset = sample_size == max(sample_size)
+    )
+  )
+
+  expect_s3_class(edist, class = "epidist")
+  expect_length(edist, 9)
```

Comment:
Why this expectation on `edist` length? The number of class members of `<epidist>` should be constant, yes?